src/main/java/com/iridium/iridiumskyblock/managers/tablemanagers/TableManager.java

```diff
@@ -62,6 +59,30 @@ public void save() {
             exception.printStackTrace();
         }
     }
+    
+    public void saveHashMap(LinkedHashMap<?, T> uuidLinkedHashMap) {
+        this.saveHashMap(uuidLinkedHashMap, true);
+    }
+    public void saveHashMap(LinkedHashMap<?, T> uuidLinkedHashMap, boolean useChanged) {
+        List<T> tList = new LinkedList<>(uuidLinkedHashMap.values());
```

Comment:
What's the point in the `t` prefix here?

Reply:
Just value is T